.github/workflows/releases.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-latest
```

Comment:
script/sign-windows-executable.sh

Reply:
Ubuntu-20.04

Reply:
It's along time to get my Windows up and running it won't let me too long in I need to take it to a commuter shop two get to the bottom of the proplhm